lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -177,6 +177,14 @@ def update_thread(request, course_id, thread_id):
     if "commentable_id" in request.POST:
         course = get_course_with_access(request.user, 'load', course_key)
         commentable_ids = get_discussion_categories_ids(course)
+        # The commentable ID on thread creation may not be available in the
+        # topic menu, such as when the thread is concerning the current
+        # unit in context. In this case the topic menu will be blank, but the
+        # old commentable_id will be sent.
```

Comment:
@antoviaque The reason why this approach was not chosen initially is that we don't glean the intended commentable_id from a normal form field on the topic selector. It's instead done by setting the topic_id on initialization, and then having that id set on the backbone object whenever the user uses the dropdown.

The dropdown itself, however, is not a form field, but a custom UI object constructed out of Divs. I could probably find a way to figure out to check if that area is blank, however it will be just as much code or more than this, since it's not a matter of 'omit if unset' so much as 'check if this UI element is on a blank element at this moment'.

Sending a commentable ID that is the current commentable ID along with the request also seems like a semantically silly thing to fail, since this commentable ID is obviously a valid ID for it to use.

Anyway, I think the test will be done soon either way. :)